LMSBorrower/src/main/java/com/lms/LMSBorrower/Controller/BorrowerController.java

```diff
@@ -1,29 +1,113 @@
 package com.lms.LMSBorrower.Controller;
 
+import java.sql.SQLException;
+import java.util.List;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Component;
 import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseStatus;
 import org.springframework.web.bind.annotation.RestController;
 
+import com.lms.LMSBorrower.POJO.Book;
 import com.lms.LMSBorrower.POJO.Borrower;
+import com.lms.LMSBorrower.POJO.LibraryBranch;
 import com.lms.LMSBorrower.Service.BorrowerService;
 
 @RestController //whole control is done through this 
-
+@Component //initializing bean
 public class BorrowerController {
 
 	@Autowired
 	BorrowerService borrowerService;
 	
-	//it gives us path; everything in {} is gonna be the input
-	//@PathVariable is for user inputs 
-	@RequestMapping("/LMSBorrower/cardNo/{cardNo}") 
+	//create new record in loans table and update copies
+	@RequestMapping(value ="/LMSBorrower/cardNo/{cardNo}/option/1/branch/{branch}/book/{book}", method = {RequestMethod.PUT, RequestMethod.GET}) 
 	
-	public Borrower getBorrowerByCardNo(@PathVariable int cardNo) {
-		return borrowerService.getBorrowerByCardNo(cardNo);
+	public  ResponseEntity<String> writeLoans(@PathVariable (value="cardNo") int cardNo, @PathVariable(value="branch") int branchId, @PathVariable (value="book") int bookId) throws SQLException {
+		boolean exists = borrowerService.ifCardExistsBorrower(cardNo);
+		
```

Comment:
These methods should be separated because they have two separate request mappings.

Reply:
one of them has been deleted since made new implementations.